r-pkg/.lintr

```diff
@@ -0,0 +1,11 @@
+linters: list(
+    "closed_curly" = lintr::closed_curly_linter
+    , "infix_spaces" = lintr::infix_spaces_linter
+    , "long_lines" = lintr::line_length_linter(length = 120)
+    , "tabs" = lintr::no_tab_linter
+    , "open_curly" = lintr::open_curly_linter
+    , "spaces_inside" = lintr::spaces_inside_linter
+    , "spaces_left_parens" = lintr::spaces_left_parentheses_linter
+    , "trailing_blank" = lintr::trailing_blank_lines_linter
+    , "trailing_white" = lintr::trailing_whitespace_linter
+ )
```

Comment:
This is great, thanks! I know that in #179 I mentioned this set of linters, but since that issue was created `lintr` has released `lintr` 2.0.0 [to CRAN](https://cran.r-project.org/web/packages/lintr/index.html) with a bunch of new ones!

Could you update the list of linters to this?

```r
LINTERS_TO_USE <- list(
    "assignment" = lintr::assignment_linter
    , "closed_curly" = lintr::closed_curly_linter
    , "equals_na" = lintr::equals_na_linter
    , "function_left" = lintr::function_left_parentheses_linter
    , "commas" = lintr::commas_linter
    , "concatenation" = lintr::unneeded_concatenation_linter
    , "implicit_integers" = lintr::implicit_integer_linter
    , "infix_spaces" = lintr::infix_spaces_linter
    , "long_lines" = lintr::line_length_linter(length = 120L)
    , "tabs" = lintr::no_tab_linter
    , "open_curly" = lintr::open_curly_linter
    , "paren_brace_linter" = lintr::paren_brace_linter
    , "semicolon" = lintr::semicolon_terminator_linter
    , "seq" = lintr::seq_linter
    , "single_quotes" = lintr::single_quotes_linter
    , "spaces_inside" = lintr::spaces_inside_linter
    , "spaces_left_parens" = lintr::spaces_left_parentheses_linter
    , "todo_comments" = lintr::todo_comment_linter
    , "trailing_blank" = lintr::trailing_blank_lines_linter
    , "trailing_white" = lintr::trailing_whitespace_linter
    , "true_false" = lintr::T_and_F_symbol_linter
)
```